webapp/channels/src/components/admin_console/system_users/system_users_date_range_selector/index.tsx

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {ReportDuration} from '@mattermost/types/reports';
+
+import {setAdminConsoleUsersManagementTableProperties} from 'actions/views/admin';
+import {getAdminConsoleUserManagementTableProperties} from 'selectors/views/admin';
+
+import * as Menu from 'components/menu';
+import Input from 'components/widgets/inputs/input/input';
+
+import './system_users_date_range_selector.scss';
+
+function get30DaysBack(now: Date) {
+    const prev = new Date(now);
+    prev.setDate(prev.getDate() - 30);
+    return prev;
```

Comment:
are we ok with this or do we need to consider timeZone as well? 0/5

Reply:
Time zone only really matters within the day, but since stats are generated daily this likely won't matter. There's a bit of drift between the two, but these values don't actually affect what is generated so should be fine.

Reply:
awesome, no worries then